### ISSUE TITLE
Resource groups showing up as multiple introductions in TagBar

### ISSUE THREAD

**Issue by codeinabox** [kylef/apiblueprint.vim#14]
I've noticed that the resource groups in the tag bar are also showing up in the TagBar as introductions. Unfortunately I don't think negative look ahead assertions are possible with Ctags.

**Comment by kylef** [comment]
@codeinabox Could you share an example blueprint please?

I'm seeing this work as expected:

<img width="927" alt="screen shot 2016-06-04 at 14 19 07" src="https://cloud.githubusercontent.com/assets/44164/15802188/50df55ae-2a5f-11e6-8db2-7492071be72a.png">

**Comment by codeinabox** [comment]
Ah but your resource groups are prefixed with `##` but I am prefixing with a single `#` as per the specification https://apiblueprint.org/documentation/specification.html#def-resourcegroup-section

**Comment by kylef** [comment]
> Ah but your resource groups are prefixed with `##` but I am prefixing with a single `#` as per the specification https://apiblueprint.org/documentation/specification.html#def-resourcegroup-section 

That's right, since my resource groups are inside the the API section (`# My API`) then they are a sub-section (`##`) inside.

The plugin should support both though.
